File: Solutions/[01]_1.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int[] data = new int[8];
		boolean ascending = true;
		boolean descending = true;
		
		for (int i = 0; i < 8; i++) {
			data[i] = sc.nextInt();
		}

		for (int i = 1; i < 8; i++) {
			if (data[i] > data[i - 1]) descending = false;
			if (data[i] < data[i - 1]) ascending = false;
		}
		
		if (ascending) System.out.println("ascending");
		else if (descending) System.out.println("descending");
		else System.out.println("mixed");
	}

}

File: Solutions/[01]_1.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	int data[8];
	bool ascending = true;
	bool descending = true;
	
	for (int i = 0; i < 8; i++) {
		cin >> data[i];
	}
	
	for (int i = 1; i < 8; i++) {
		if (data[i] > data[i - 1]) descending = false;
		if (data[i] < data[i - 1]) ascending = false;
	}
	
	if (ascending) cout << "ascending";
	else if (descending) cout << "descending";
	else cout << "mixed";
}
